frontend/src/content/news.json

```diff
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-04-04",
+      "title": "Metabolic Atlas 2.5 improves the about pages and fixes bugs",
+      "text": "<b>Metabolic Atlas 2.5</b> is out. In this minor release we have focused on maintenance, fixing a number of bugs related to search and routing. General improvements for the content pages, including the <i>About</i> pages and <i>Documentation</i>, have been added as well. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.5' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Comment:
`routing` sounds too dev oriented for the news, are there any other ideas?

Reply:
`routing` is the reasoning of the bug and in the news I think it is better to describe the bug itself. What about `broken links` or `broken URLs`?

Reply:
`broken links` should be great, thanks! Addressed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/942/commits/4f9a85ada008534320266fc9ac61b4eb90da681a